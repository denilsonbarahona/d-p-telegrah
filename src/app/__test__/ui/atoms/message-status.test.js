import React from "react";
import { render } from "@testing-library/react";
import MessageStatus from "@atoms/messageStatus";

describe("describe", () => {
  it("rendering without crashing", () => {
    const { getByText } = render(<MessageStatus time="today" />);
    expect(getByText("today")).toBeInTheDocument();
  });

  it("to match snapshot", () => {
    const { container } = render(<MessageStatus time="today" />);
    expect(container).toMatchSnapshot();
  });
});
