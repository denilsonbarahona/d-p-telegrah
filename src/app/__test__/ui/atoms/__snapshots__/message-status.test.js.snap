// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`describe to match snapshot 1`] = `
<div>
  <div
    class="sc-bczRLJ eGOsWw"
  >
    <svg
      fill="currentColor"
      height="1em"
      stroke="currentColor"
      stroke-width="0"
      viewBox="0 0 512 512"
      width="1em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M464 128L240 384l-96-96m0 96l-96-96m320-160L232 284"
        fill="none"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="32"
      />
    </svg>
    <p
      class="sc-gsnTZi cJoxYx"
    >
      today
    </p>
  </div>
</div>
`;
